boards/common/samd21-arduino-bootloader/Makefile

```diff
@@ -0,0 +1,3 @@
+MODULE = boards_common_samd21-arduino-bootloader
```

Comment:
```Suggestion
MODULE = boards_common_samd21_arduino-bootloader
```
?

Reply:
The module name is `samd21-arduino-bootloader` so I think the name is ok.